src/cargo/util/toml/mod.rs

```diff
@@ -1530,25 +1530,23 @@ fn unique_build_targets(
 }
 
 #[derive(Debug, Deserialize, Serialize, Clone)]
+#[serde(rename_all = "kebab-case")]
 pub struct TomlWorkspace {
     members: Option<Vec<String>>,
-    #[serde(rename = "default-members")]
-    default_members: Option<Vec<String>>,
     exclude: Option<Vec<String>>,
+    default_members: Option<Vec<String>>,
     resolver: Option<String>,
+    metadata: Option<toml::Value>,
 
     // Properties that can be inherited by members.
     package: Option<InheritableFields>,
     dependencies: Option<BTreeMap<String, TomlDependency>>,
     lints: Option<TomlLints>,
-
-    // Note that this field must come last due to the way toml serialization
-    // works which requires tables to be emitted after all values.
```

Comment:
Seems like metadata isn't needed to be the last anymore. Do we have a test to verify the change?

Reply:
Looks like we have several tests in `testsuite/package.rs` that use `package.metadata` which means we'll serialize the prepared manifest and verify its output which would hit this situation.

Reply:
This was <https://github.com/rust-lang/cargo/commit/6bc74556f9dbfe6d50a0e51a20591f420fb72c0a> but since we have tests and `toml` crate has changed largely. I am going to merge this. Thank you!